astropy/io/votable/table.py

```diff
@@ -61,10 +60,6 @@ def parse(source, columns=None, invalid='exception', verify=None,
         for more information. When not provided, uses the configuration setting
         ``astropy.io.votable.verify``, which defaults to 'ignore'.
 
-        .. versionchanged:: 4.0
```

Comment:
Keep this note, at least until `pedantic` fully disappears. I would even argue that the `versionadded` info, without the extra explanation is a very useful thing.

Reply:
Same comment as @bsipocz .